index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/styles.css" />
+    <script type="module" src="scripts/index.js"></script>
+    <title>Haker_news_JS</title>
+  </head>
+  <body>
+    <div class="container">
+      <header class="container__header">
+        <div class="logo">
+          <div class="logo__image"><img src="images/logo.gif" /></div>
+          <h1 class="logo__title">Hacker News</h1>
+        </div>
+        <nav>
+          <ul class="menu">
+            <li class="menu__item">
+              <a id="user" href="" class="">User info</a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+      <div class="container__body">
+        <div>
+          <ol id="news-list" type="1" class="generic-list">
+          </ol>
+        </div>
+        <div>
+          <form class="form" name="user">
+            <h2 class="form__title">Login form</h2>
+            <ul id="formMistake" class="form__mistake">   </ul>
+            <div class="form__item">
```

Comment:
I would say form element is not an 'item' but a control or a field. In addition we could decompose form block by itself from its controls. So that form block is responsible for the positioning of it's elements only. The way how an element looks belongs to field block:

`
.form {...}
.field {...}
`    
<div class="form__control field">

Reply:
Fixed